Deployment Guides/Provisioning-Islandora-on-Ubuntu.md

```diff
@@ -120,280 +129,279 @@ cat ~/islandora-install.properties
 
-<code>apt-get -y install oracle-java7-installer libjpeg-dev libpng12-dev libtiff4-dev php5 php5-cli php5-curl php5-dev php5-gd php5-ldap php5-mysql php5-xsl php-apc php-soap php-xml-htmlsax3 php-xml-parser php-xml-rpc php-xml-rpc2 php-xml-rss php-xml-serializer php5-imagick php5-mcrypt php-xml* mysql-server vim curl apache2 rsync wget imagemagick ant libimage-exiftool-perl unzip lame autoconf build-essential checkinstall git libass-dev libfaac-dev libgpac-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev librtmp-dev libtheora-dev libtool libvorbis-dev pkg-config texi2html zlib1g-dev ffmpeg2theora poppler-utils python-pip libreoffice libreoffice-writer libreoffice-calc libreoffice-impress libreoffice-draw bibutils ufraw links monit tesseract-ocr tesseract-ocr-eng tesseract-ocr-fra tesseract-ocr-spa tesseract-ocr-ita tesseract-ocr-por tesseract-ocr-hin tesseract-ocr-deu tesseract-ocr-jpn tesseract-ocr-rus</code>
+```
+apt-get -y install oracle-java7-installer libjpeg-dev libpng12-dev libtiff4-dev php5 php5-cli php5-curl php5-dev php5-gd php5-ldap php5-mysql php5-xsl php-apc php-soap php-xml-htmlsax3 php-xml-parser php-xml-rpc php-xml-rpc2 php-xml-rss php-xml-serializer php5-imagick php5-mcrypt php-xml* mysql-server vim curl apache2 rsync wget imagemagick ant libimage-exiftool-perl unzip lame autoconf build-essential checkinstall git libass-dev libfaac-dev libgpac-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev librtmp-dev libtheora-dev libtool libvorbis-dev pkg-config texi2html zlib1g-dev ffmpeg2theora poppler-utils python-pip libreoffice libreoffice-writer libreoffice-calc libreoffice-impress libreoffice-draw bibutils ufraw links monit tesseract-ocr tesseract-ocr-eng tesseract-ocr-fra tesseract-ocr-spa tesseract-ocr-ita tesseract-ocr-por tesseract-ocr-hin tesseract-ocr-deu tesseract-ocr-jpn tesseract-ocr-rus
+```
 
-### Software Dependencies compiled from Source  
+### Software Dependencies Compiled from Source <a id="from-source"></a> 
 
-<b>ghostscript</b>  
+#### ghostscript  <a id="ghostscript"></a>
 
-_Please note: ubuntu 14.04 installs ghostscript 9.10 which is currently failing our test sets and prevents tiffs from being generated from pdfs. Recommend installing from source to get correct version_
+**Notes:** ubuntu 14.04 installs ghostscript 9.10 which is currently failing our test sets and prevents tiffs from being generated from pdfs. Recommend installing from source to get correct version.
 
-<code>wget http://downloads.ghostscript.com/public/ghostscript-9.05.tar.gz</code>
+`wget http://downloads.ghostscript.com/public/ghostscript-9.05.tar.gz`
 
-<code>tar xvzf ghostscript-9.05.tar.gz</code>
+`tar xvzf ghostscript-9.05.tar.gz`
 
-<code>cd ghostscript-9.05</code>
+`cd ghostscript-9.05`
   
-<code>./configure</code>
+`./configure`
  
-<code>make && make install</code> 
+`make && make install`
 
+`gs --version` should return result.
 
-_should return result_  
+#### ffmpeg  <a id="ffmpeg"></a>
```

Comment:
I think this was more of a markup typo on my part. Probably don't need that heading. The first part is just making a folder to put all the ffmpeg related dependancy stuff in. The second part is the actual ffmpeg build.